pre_commit/languages/r.py

```diff
@@ -88,13 +88,11 @@ def install_environment(
     env_dir = _get_env_dir(prefix, version)
     with clean_path_on_failure(env_dir):
         os.makedirs(env_dir, exist_ok=True)
-        path_desc_source = prefix.path('DESCRIPTION')
-        if os.path.exists(path_desc_source):
-            shutil.copy(path_desc_source, env_dir)
         shutil.copy(prefix.path('renv.lock'), env_dir)
         cmd_output_b(
             'Rscript', '--vanilla', '-e',
-            """\
+            "prefix_dir <- '" + prefix.prefix_dir + "'" +
```

Comment:
this should use an f-string to do formatting, also triple-quoted literals can contain quote characters

the backslash was intentional, actually -- it makes the line numbers of the string match up with the string (and avoids a blank line at the top of your string)

Reply:
Thanks, makes all sense. Was too lazy to check if `python < 3.6` is required to make f strings work... should be fixed now. Also, because the `prefix_dir` is passed into string expression (`Rscript -e {expr}`), another escaping is required for Windows, as it has backward slashes in the path. Maybe there is a better way to do this but I could not find one...